apps/testingaccessibility/src/components/app/navigation.tsx

```diff
@@ -125,7 +128,7 @@ const MobileNav: React.FC = () => {
                   )}
                 </Menu.Item>
               </div>
-              {!isLoadingUser && isSellingLive && (
+              {userLoadingStatus !== 'loading' && isSellingLive && (
```

Comment:
checking the status directly instead of using the helper since "what can they do" is different than "when can they do it"